src/cmd-coreos-prune

```diff
@@ -411,10 +411,7 @@ def prune_images(s3, build, images_to_keep, dry_run, bucket, prefix):
                     s3.delete_object(Bucket=bucket, Key=image_prefix)
                     print(f"\t\t\tPruned {name} image for {build.id} for {build.arch}")
                 except botocore.exceptions.ClientError as e:
-                    if e.response['Error']['Code'] == 'NoSuchKey':
-                        print(f"\t\t\t{bucket}/{image_prefix} already pruned.")
-                    else:
-                        errors.append(e)
```

Comment:
Hmm, feels like it's still worth a comment somewhere in this area that the API returns 204 in the `ENOENT` case (and/or link to the docs page). As someone reviewing this code, I wouldn't have guessed that so my first thought would've been that it's a bug.

Reply:
added a comment